ecs-run-task/run_task.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e -u
+
+# Run an ECS Task from the provided CLUSTER and SERVICE
+#
+# This actions is generally used for ad-hoc launching of ECS tasks that are scheduled with EventBridge
```

Comment:
question(non-blocking): For my own learning, how are these tasks stopped?

Reply:
Generally, they are designed to not run continuously. The service starts, does some work and then exits when the work is complete. They are kind of like long running lambda functions.